lib/geoengineer/resources/aws_sfn_state_machine_spec.rb

```diff
@@ -0,0 +1,28 @@
+########################################################################
+# AwsSfnStateMachine is the +aws_sfn_state_machine+ terrform resource,
```

Comment:
Mean to put a period rather than a comma?

Reply:
No idea.